### PR TITLE
DatePickerIOS - Added Snack Examples 🍿

### PR DESCRIPTION
**Pull request by DaniAkash**
Belongs to  facebook/react-native-website#1579

Created a nice little snack for people to view DatePickerIOS in action using both Function & Class Component examples.

### CODE REVIEW

File: docs/datepickerios.md [hunk 1]
```diff
@@ -9,9 +9,50 @@ Use `DatePickerIOS` to render a date/time picker (selector) on iOS. This is a co
...
+      Class Component Example
+    </li>
+  </ul>
+</div>
+
+<block class="functional syntax" />
+
+```SnackPlayer name=DatePickerIOS&supportedPlatforms=ios
+import React, {useState} from 'react';
+import {DatePickerIOS, View, StyleSheet, Text, Platform} from 'react-native';
```
Comment (Simek) [inline docs/datepickerios.md]: `Platform` and `Text` imports are unnecessary after af16849a7ffc97bb402c117867385cd2084d09a0 commit.

File: docs/datepickerios.md [hunk 2]
```diff
@@ -9,9 +9,50 @@ Use `DatePickerIOS` to render a date/time picker (selector) on iOS. This is a co
...
+  },
+});
+```
+
+<block class="classical syntax" />
+
+```SnackPlayer name=DatePickerIOS&supportedPlatforms=ios
 import React, {Component} from 'react';
-import {DatePickerIOS, View, StyleSheet} from 'react-native';
+import {DatePickerIOS, View, StyleSheet, Text, Platform} from 'react-native';
```
Comment (Simek) [inline docs/datepickerios.md]: `Platform` and `Text` imports are unnecessary after af16849a7ffc97bb402c117867385cd2084d09a0 commit.

**Comment by Simek** [comment]
I'm not sure if adding an example for the the deprecated components is a good idea. 

They will be sooner or later completely removed form the codebase and not providing examples for them could be a good way to discourage usage of those components.

File: docs/datepickerios.md (hunk 1 above)
Comment (DaniAkash) [reply docs/datepickerios.md]: Have removed these imports

File: docs/datepickerios.md (hunk 2 above)
Comment (DaniAkash) [reply docs/datepickerios.md]: have removed them

**Comment by DaniAkash** [comment]
> I'm not sure if adding an example for the the deprecated component is a good idea.
> 
> They will be sooner or later completely removed form the codebase and not providing examples for them could be a good way to discourage usage of those components.

@Simek I'll leave the decision to the maintainers 👍

**Comment by Simek** [comment]
It seems that Rachel got similar concerns to my: https://github.com/facebook/react-native-website/issues/1579#issuecomment-582448869

Also after thinking about this a bit more I would like to start discussion about removing all examples form the deprecated components after #1493 will be merged.

**Comment by rachelnabors** [comment]
> Also after thinking about this a bit more I would like to start discussion about removing **all** examples from the deprecated components after #1493 will be merged and before next major version (0.61 docs and below will remain untouched).

Actually we want to remove all the deprecated components from the sidebar for 0.62. Want to do it for me?

**Comment by rickhanlonii** [comment]
Keep in mind https://github.com/facebook/react-native-website/issues/1045 while removing pages - if people have bookmarks or page ranked search results we should guide them to the new location for awhile.

**Comment by Simek** [comment]
@rickhanlonii I think that @rachelnabor want only to remove  entries for the deprecated components from the Sidebar and leave pages intact under versioned `0.62` urls.

@rachelnabors I can help with that, no problem. It should help if you describe exactly what is desirable effect of that change. Currently I'm assuming that you want me to to generate versioned docs for `0.62` (they do not exist ATM) and remove those pages ONLY form the Sidebar in there. Is that right?

**Comment by rachelnabors** [comment]
@rickhanlonii So @Simek has me right here—would be great to just remove the links from the sidebar from 62 onward. Editing only **sidebars.json** should change this in **/docs/next/** and all versions going forward—which is what we want! I'd submit a PR for that very thing, @Simek :)

**Comment by rickhanlonii** [comment]
Yup that works.

**Pull request merged**